tools/runner/logsaver.go

```diff
@@ -28,33 +28,37 @@ func NewLogSaver(podsGetter corev1types.PodsGetter) *LogSaver {
 	}
 }
 
-// SavePodLogs saves pod logs to files with same name as pod.
-func (ls *LogSaver) SavePodLogs(ctx context.Context, loadTest *grpcv1.LoadTest, podLogDir string) error {
+// SavePodLogs saves pod logs to files with same name as pod. It returns a map
```

Comment:
The first line in the godoc should stand by itself, because it is extracted for documentation.

Breaking after the period and writing something like "This function returns a map..." should be enough.

You may want to keep these lines within 80 columns.

Reply:
Done. I'm going to spend a few minutes looking at godoc so I understand how docs are generated 👍